### ISSUE TITLE
Cannot Access Pattern Storage

### ISSUE THREAD

**Issue by rellis32** [TechReborn/RebornStorage#69]
Just updated to the latest version on Direwolf modpack and now cannot access reborn storage to add or remove patterns. Getting this in debug of console whenever I right-click the structure:

10:24:11  [Info] [STDOUT]: [me.modmuss50.rebornstorage.tiles.TileMultiCrafter:getValidLastPage:175]: readfalse
10:24:11  [Error] [FML]: FMLIndexedMessageCodec exception caught

This is a 7X7 structure that was working fine prior to update.

Minecraft v. 1.12.2
Reborn Core v.3.5.1.166
RebornStorage v.3.0.1.26

**Comment by neo130288** [comment]
iam having the same issue, evan tried to hopper the new patterns in no luck

**Comment by modmuss50** [comment]
can you give me the whole log please?

**Comment by frozenbrain** [comment]
```[17:53:05] [Server thread/INFO] [STDOUT]: [me.modmuss50.rebornstorage.tiles.TileMultiCrafter:getValidLastPage:175]: readfalse
[17:53:05] [Server thread/ERROR] [FML]: FMLIndexedMessageCodec exception caught
java.lang.RuntimeException: Missing
	at net.minecraftforge.fml.server.FMLServerHandler.getClientToServerNetworkManager(FMLServerHandler.java:298) ~[FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.getClientToServerNetworkManager(FMLCommonHandler.java:553) ~[FMLCommonHandler.class:?]
	at net.minecraftforge.fml.common.network.FMLOutboundHandler$OutboundTarget$8.selectNetworks(FMLOutboundHandler.java:248) ~[FMLOutboundHandler$OutboundTarget$8.class:?]
	at net.minecraftforge.fml.common.network.FMLOutboundHandler.write(FMLOutboundHandler.java:297) ~[FMLOutboundHandler.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:738) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:730) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:816) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:723) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:111) ~[MessageToMessageEncoder.class:?]
	at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:738) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:801) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:814) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:794) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:831) ~[AbstractChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1032) ~[DefaultChannelPipeline.class:?]
	at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:296) ~[AbstractChannel.class:?]
	at net.minecraftforge.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToServer(SimpleNetworkWrapper.java:289) [SimpleNetworkWrapper.class:?]
	at reborncore.common.network.NetworkManager.sendToServer(NetworkManager.java:60) [NetworkManager.class:?]
	at me.modmuss50.rebornstorage.blocks.BlockMultiCrafter.func_180639_a(BlockMultiCrafter.java:83) [BlockMultiCrafter.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:446) [or.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:739) [pa.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:55) [ma.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:11) [ma.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_101]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:721) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396) [nz.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]

**Comment by modmuss50** [comment]
Humm, im not sure what is going on there. I will have to take a deeper look. Thanks for opening an issue.

**Comment by neo130288** [comment]
mine is a wee bit longer and only in the fml-junk-earlystartup.log so paste bined it:

https://pastebin.com/ZMjzkLy2

also it still function just cant add new patterns 
 
hope this helps

**Comment by DartEtt** [comment]
same problem, but i cant enter to structure, but the same structure works normal in creative(5x5). and when crafting the structure **Block is not valid for use in the machine's interior**  was error report in chat, after finishing structure in creative all worked well, on server same error and multiblock not working.

**Comment by modmuss50** [comment]
humm, thanks for that. That should help me track the issue down.

**Comment by kery2000** [comment]
it keeps putting out that the heat conductor on the top center of the 3x3x4 is not valid for use in the machine

**Comment by modmuss50** [comment]
Are you guys on a multiplayer server?

**Comment by kery2000** [comment]
Yes

**Comment by modmuss50** [comment]
Nice, thanks I can confirm that is an issue on multiplayer servers.

**Comment by kery2000** [comment]
Good to know hope its something easy to fix

**Comment by romulous75** [comment]
+1. I'm having the same issue.

**Comment by boballama** [comment]
This issue seems to be marked as closed but it gives no indication of the problem being corrected.  Neither does the Reborn Storage mod's website and/or the DW20 file list.  But it does appear to be fixed.  I downloaded the updated version of Reborn Storage, Reborn Core, Refined Storage and MCMultiPart to the versions included in the 1.4.0 beta version of this modpack and the RebornStorage multiblock now opens correctly--Yippee!  Note:  The 1.3.0 version of this modpack uses the same versions of these files as noted by the OP.  (I chose not to download the entire 1.4.0 beta version of this modpack.)
